static/app/components/arithmeticBuilder/token/index.spec.tsx

```diff
@@ -0,0 +1,91 @@
+import {render, screen} from 'sentry-test/reactTestingLibrary';
+
+import type {Token} from 'sentry/components/arithmeticBuilder/token';
+import {
+  TokenFreeText,
+  TokenOperator,
+  TokenParenthesis,
+} from 'sentry/components/arithmeticBuilder/token';
+import {TokenGrid} from 'sentry/components/arithmeticBuilder/token/grid';
+import {
+  tokenizeExpression,
+  toOperator,
+  toParenthesis,
+} from 'sentry/components/arithmeticBuilder/tokenizer';
+
+function k<T extends Token>(token: T): T {
+  token.key = expect.any(String);
+  return token;
+}
+
+function s(value: string): TokenFreeText {
+  return k(new TokenFreeText(expect.any(Object), value));
+}
+
+function o(value: string): TokenOperator {
+  return k(new TokenOperator(expect.any(Object), toOperator(value)));
+}
+
+function p(value: string): TokenParenthesis {
+  return k(new TokenParenthesis(expect.any(Object), toParenthesis(value)));
+}
+
+describe('token', function () {
+  describe('ArithmeticTokenFreeText', function () {});
+
+  describe('ArithmeticTokenFunction', function () {});
```

Comment:
```suggestion
```
Unless you plan to add these

Reply:
Leaving these empty for now, going to add them later.